inception-imls-dl4j/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/dl4j/pos/DL4JSequenceRecommenderTest.java

```diff
@@ -236,7 +241,8 @@ public void testExtractOverlappingLabelsFails2() throws Exception
     @Test
     public void thatPosTrainingWorks() throws Exception
     {
-        DL4JSequenceRecommender sut = new DL4JSequenceRecommender(buildPosRecommender(), traits);
+        DL4JSequenceRecommender sut = new DL4JSequenceRecommender(buildPosRecommender(), traits,
```

Comment:
You could just write a function which creates the dl4j instead of copying this always around.